### ISSUE TITLE
Not playing songs

### ISSUE THREAD

**Issue by arlexTech** [museofficial/muse#1065]
Hello, i deployed a container with

```
 services:
  muse:
    image: codetheweb/muse:latest
    restart: always
    volumes:
      - /config/muse:/data
    environment:
      - DISCORD_TOKEN=xxxxxxxxxxxx
      - YOUTUBE_API_KEY=xxxxxxxxxxxxx
      - SPOTIFY_CLIENT_ID=xxxxxxxxxxxxxx
      - SPOTIFY_CLIENT_SECRET=xxxxxxxxxxxxxx
      - DEBUG=*
```
but when i use ```/play query:https://youtu.be/CqnU_sJ8V-E```, it join the channel and post the message but then nothing happens and it disconnect after few seconds
![image](https://github.com/user-attachments/assets/e3297916-f8e4-46c8-9040-51fd80fcf2d0)
[_muse-muse-1_logs.txt](https://github.com/user-attachments/files/16626936/_muse-muse-1_logs.txt)

thank you for your help

Edit:

it work if i use ```/play query:YouTube: free bird``` it works but play another version

![image](https://github.com/user-attachments/assets/826eb566-1397-4548-8056-e81e4fcb2fca)

it works for some links but not all

**Comment by shiftybitshiftr** [comment]
Can you try the image `codetheweb/muse:pr-1066` and see if that fixes it?

**Comment by DomiiBunn** [comment]
I can confirm that it does

**Comment by matiasba** [comment]
Same, that pr fix my bot

**Comment by HellboundGlory** [comment]
How do I try the image? I have the same problem I think

**Comment by DomiiBunn** [comment]
Think this can be closed as it's basically a dupe of #1055 and the fix is to update the image to `ghcr.io/museofficial/muse:latest`
